Ladybird/AppKit/UI/Tab.mm

```diff
@@ -195,6 +195,10 @@ - (TabController*)tabController
 
 - (void)updateTabTitleAndFavicon
 {
+    CGFloat titleFontSize = 12;
+    CGFloat faviconSize = 16;
```

Comment:
Last comment - let's keep these in line with the coding style guide:
https://github.com/LadybirdBrowser/ladybird/blob/master/Documentation/CodingStyle.md#names

```c++
static constexpr CGFloat TITLE_FONT_SIZE = 12;
static constexpr CGFloat FAVICON_SIZE = 16;
```